src/amo/components/UserProfile/index.js

```diff
@@ -162,7 +162,7 @@ export class UserProfileBase extends React.Component<Props> {
                 addonType={ADDON_TYPE_EXTENSION}
                 authorDisplayName={[user.displayName]}
                 authorUsernames={[user.username]}
-                numberOfAddons={3}
+                numberOfAddons={10}
```

Comment:
Sorry for not noticing this before, but you've introduced a `EXTENSIONS_BY_AUTHORS_PAGE_SIZE` in the reducer, so we might as well use that value here as well.